src/components/ManagerPage/AppsList.js

```diff
@@ -275,17 +324,45 @@ class AppsList extends PureComponent<Props, State> {
           textAlign="center"
           style={{ maxWidth: 350 }}
         >
-          {t(`manager.apps.${mode === 'installing' ? 'installSuccess' : 'uninstallSuccess'}`, {
-            app,
+          {t(`manager.apps.${isInstalling ? 'installSuccess' : 'uninstallSuccess'}`, {
+            app: get(app, 'name', ''),
           })}
         </Box>
+        {app && app.currency && isInstalling ? (
+          <Box
+            color="smoke"
+            mt={2}
+            fontSize={4}
+            ff="Open Sans|Regular"
+            textAlign="center"
+            style={{ maxWidth: 350 }}
+          >
+            {t(`manager.apps.${isSupported ? 'supportedCurrency' : 'unsupportedCurrency'}`, {
+              app: app.name,
+              device: get(getDeviceModel(device.modelId), 'productName', ''),
+            })}
+          </Box>
+        ) : null}
       </Box>
     ) : null
   }
 
   renderFooter = () => {
     const { t } = this.props
-    return (
+    const { app, mode } = this.state
+
+    const isInstalling = mode === 'installing'
+    const isCurrency = app && app.currency
+    const isSupported = app && supported.includes(app.name.toLowerCase())
```

Comment:
this does not seem correct to do `app.name.toLowerCase()` here. app name does not match app names but instead should match `app.currency.id`

so basically we should do this

```
isSupported = app && app.currency && isCurrencySupported(app.currency)
```

and probably introduce this `isCurrencySupported` instead of exporting `supported`